docs/pages/install/docker.mdx

```diff
@@ -140,3 +140,38 @@ It is recommended that admins bind a Pelican configuration file on the host mach
 
 
 4. You should note that your running Pelican origin server is logging debug level messages.
+
+## Additional Configurations for Running Pelican in a Container
+
+### Server hostname and port
+
+Pelican server has a built-in web server to handle various API requests. By default, it uses server hostname and a port number as the server address (e.g. `https://127.0.0.1:8444`). In a container environment, the hostname is an alphanumeric value, e.g. `6ee28c5df997`, and the corresponding Pelican server address is `https://6ee28c5df997:8444`. This value is for container internal communications and is not accessible from outside of the container. In local deployment, to access the container ports from the host machine, one can publish a port from inside the container to the host machine.
```

Comment:
Perhaps link out to the Docker network documentation (https://docs.docker.com/network/) for more information on how the Docker networking works?